src/modules/search/components/menu/types/geoResource/GeoResourceResultItem.js

```diff
@@ -144,18 +165,18 @@ export class GeoResourceResultItem extends MvuElement {
 		const getZoomToExtentButton = (result) => {
```

Comment:
Do not show the zoom-to-extent button until the GeoResource is temporarily added to the map